### PR TITLE
updating elasticsearch download links & version

### PR DESCRIPTION
**Pull request by li3n3**
Also some little grammar things, while I'm at it!

### CODE REVIEW

File: jekyll/_docs/installing-elasticsearch.md [hunk 1]
```diff
@@ -4,41 +4,41 @@ title: Install a custom version of Elasticsearch
...
+    - wget https://download.elastic.co/elasticsearch/release/org/elasticsearch/distribution/tar/elasticsearch/2.4.3/elasticsearch-2.4.3.tar.gz
+    - tar -xvf elasticsearch-2.4.3.tar.gz
+    - elasticsearch-2.4.3/bin/elasticsearch: {background: true}
     # Make sure that Elasticsearch is up before running tests:
     - sleep 10 && wget --waitretry=5 --retry-connrefused -v http://127.0.0.1:9200/
 ```
 
 <span class='label label-info'>Note:</span>
-remember to remove elasticsearch from machine.services if you install it manually.
+remember to remove elasticsearch from `machine.services` if you install it manually.
```
Comment (li3n3) [inline jekyll/_docs/installing-elasticsearch.md]: Is `machine.services` the correct way to refer to this section in `circle.yml`?

**Comment by li3n3** [comment]
I would also love a quick check to make sure that my updating of links is consistent/good.

File: jekyll/_docs/installing-elasticsearch.md (hunk 1 above)
Comment (felicianotech) [inline jekyll/_docs/installing-elasticsearch.md]: I don't normally do it that way but I think I like your way better. I think maybe somewhere in parentheses mentioned `circle.yml`?

File: jekyll/_docs/installing-elasticsearch.md [hunk 2]
```diff
@@ -4,41 +4,41 @@ title: Install a custom version of Elasticsearch
...
 
 ```
 machine:
   services:
     - elasticsearch
 ```
 
-The default version of elasticsearch is {{ site.data.precise.versions.elasticsearch }}.
-If you need a custom version, you can download and start it from your build. To install 1.0.1, add the following to your circle.yml:
+The default version of Elasticsearch is {{ site.data.precise.versions.elasticsearch }}.
```
Comment (drazisil) [inline jekyll/_docs/installing-elasticsearch.md]: Wherever https://circleci.com/docs/build-image-trusty/#elasticsearch comes from

File: jekyll/_docs/installing-elasticsearch.md (hunk 1 above)
Comment (levlaz) [inline jekyll/_docs/installing-elasticsearch.md]: @li3n3 I think `machine: services:` is a bit more straightforward.

Comment (li3n3) [inline jekyll/_docs/installing-elasticsearch.md]: Thanks! This was the first time I'd seen this way of writing it, and I didn't know if it was some standard secret dot notation I didn't know about. Will change.

Comment (li3n3) [inline jekyll/_docs/installing-elasticsearch.md]: Linked to that doc as well, @drazisil

File: jekyll/_docs/installing-elasticsearch.md (hunk 2 above)
Comment (li3n3) [inline jekyll/_docs/installing-elasticsearch.md]: If I just s/precise/trusty, will that do the trick? (Or how would I test that?)

Comment (drazisil) [inline jekyll/_docs/installing-elasticsearch.md]: I would change to trusty, and maybe add a tad of text stating thats the image you are talking about. Remember we don't care about 12.04 anymore :)

**Comment by zzak** [comment]
LGTM once it's green

**Comment by zzak** [comment]
LGTM once it's green

File: jekyll/_docs/installing-elasticsearch.md [hunk 3]
```diff
@@ -4,41 +4,41 @@ title: Install a custom version of Elasticsearch
...
+    - wget https://download.elastic.co/elasticsearch/release/org/elasticsearch/distribution/tar/elasticsearch/2.4.3/elasticsearch-2.4.3.tar.gz
+    - tar -xvf elasticsearch-2.4.3.tar.gz
+    - elasticsearch-2.4.3/bin/elasticsearch: {background: true}
     # Make sure that Elasticsearch is up before running tests:
     - sleep 10 && wget --waitretry=5 --retry-connrefused -v http://127.0.0.1:9200/
 ```
 
 <span class='label label-info'>Note:</span>
-remember to remove elasticsearch from machine.services if you install it manually.
+remember to remove elasticsearch from [`machine: services:`]({{site.baseurl}}configuration/#services/) if you install it manually.
```
Comment (felicianotech) [inline jekyll/_docs/installing-elasticsearch.md]: The link should be: `[`machine: services:`]({{site.baseurl}}/configuration/#services)`.

File: jekyll/_docs/installing-elasticsearch.md (hunk 2 above)
Comment (li3n3) [inline jekyll/_docs/installing-elasticsearch.md]: Current version of this page pulls in 0.90.2 from Precise.

File: jekyll/_docs/installing-elasticsearch.md (hunk 3 above)
Comment (li3n3) [inline jekyll/_docs/installing-elasticsearch.md]: Awesome; thanks. Am excited about relative links but haven't been building them lately!

**Comment by li3n3** [comment]
Huzzah — green build! Would love a ✔️.

**Pull request merged**